stablehlo/reference/Ops.h

```diff
@@ -38,6 +38,9 @@ Tensor evalNotOp(const Tensor &operand, Type resultType);
 Tensor evalOrOp(const Tensor &lhs, const Tensor &rhs, Type resultType);
 Tensor evalReshapeOp(const Tensor &operand, Type resultType);
 Tensor evalSineOp(const Tensor &operand, Type resultType);
+Tensor evalSliceOp(const Tensor &operand,
+                   const SmallVector<int64_t> startIndices,
```

Comment:
Apologies - I'm just noticing this. This should be `ArrayRef<int64_t>` instead of `SmallVector<int64_t>`.

Reply:
Thanks!